survey/3-tech.yml

```diff
@@ -137,14 +137,16 @@ questions:
       - Next
       - Other
 
-  - label: What are the platforms and tools you are using on a daily basis?
+  - label: "What are the platforms and tools you are using on a daily basis?"
     required: true
     multiple: true
     choices:
       - Node.js
       - .NET
       - Pandas
       - TensorFlow
+      - Docker
+      - Kubernetes
```

Comment:
Could we consider hashicorp tools a platform ?

Reply:
Sure, Terraform is one of them, what would you add?